### ISSUE TITLE
Can't initialize infinitescroll

### ISSUE THREAD

**Issue by kedves** [abecks/meteor-infinite-scroll#3]
Hi.

After setting up the package based on the description, I got an error which says that i need `app` .

I made a global `app = Meteor;` but still doesn't work.

```
TypeError: Cannot read property 'Events' of undefined
    at infiniteScroll (infinite-scroll.js:76)
    at Template.home.rendered (home.js:12)
    at template.js:116
    at Function.Template._withTemplateInstanceFunc (template.js:457)
    at fireCallbacks (template.js:112)
    at .<anonymous> (template.js:205)
    at view.js:107
    at Object.Blaze._withCurrentView (view.js:538)
    at view.js:106
    at Object.Tracker._runFlush (tracker.js:497)
```

**Comment by lucascurti** [comment]
Same problem here.

**Comment by leizard** [comment]
Hi, is this fixed ?

**Comment by abecks** [comment]
Should be fixed now with #5
